python/tvm/relay/prelude.py

```diff
@@ -534,6 +538,22 @@ def __init__(self, mod):
         self.define_tree_map()
         self.define_tree_size()
 
-        self.define_id()
-        self.define_compose()
         self.define_iterate()
+
+__PRELUDE_PATH__ = os.path.dirname(os.path.realpath(__file__))
+
+def load_prelude(mod=None):
+    '''Parses the portions of the Prelude written in Relay's text format and adds
```

Comment:
newline after `'''`. Consider using either `'''` or `"""` for all docstrings in this file

Reply:
Done